secboot/secboot_sb_test.go

```diff
@@ -32,9 +32,14 @@ import (
 	"os"
 	"path/filepath"
 
+	. "gopkg.in/check.v1"
+
 	"github.com/canonical/go-tpm2"
+	"github.com/canonical/go-tpm2/linux"
+
 	sb "github.com/snapcore/secboot"
-	. "gopkg.in/check.v1"
+	sb_efi "github.com/snapcore/secboot/efi"
+	sb_tpm2 "github.com/snapcore/secboot/tpm2"
 
```

Comment:
this should make the linter happy again:

```
diff --git a/secboot/secboot_sb_test.go b/secboot/secboot_sb_test.go
index 8388067634..6cbc50bb89 100644
--- a/secboot/secboot_sb_test.go
+++ b/secboot/secboot_sb_test.go
@@ -32,14 +32,12 @@ import (
        "os"
        "path/filepath"
 
-       . "gopkg.in/check.v1"
-
        "github.com/canonical/go-tpm2"
        "github.com/canonical/go-tpm2/linux"
-
        sb "github.com/snapcore/secboot"
        sb_efi "github.com/snapcore/secboot/efi"
        sb_tpm2 "github.com/snapcore/secboot/tpm2"
+       . "gopkg.in/check.v1"
 
        "github.com/snapcore/snapd/asserts"
        "github.com/snapcore/snapd/asserts/assertstest"
```

Reply:
but @anonymouse64 told me to do the reverse. i am confused.

Reply:
I can't seem to find what I told you, but I think I probably said something to the effect of "separate imports with external coming first, then internal", and the issue you have is that you may have conflated my saying "external" with "external to canonical/snapcore org" when really I meant external as external to the snapd repo, so we have this as the desired pattern:
```go
import (
    "standard library from go like fmt"

    "external from the snapd repo like github.com/snapcore/secboot or github.com/canonical/efilib or gopkg.in/gocheck.v1" 

    "internal to snapd repo like github.com/snapcore/snapd/osutil/..."
)
```

does that make sense ?

Reply:
it's also entirely possible that I told you the inverse and I told you to put internal deps first and external second, but if I said that I was wrong, and I apologize for the confusion